src/Data/Traversable.purs

```diff
@@ -60,6 +62,39 @@ sequenceDefault :: forall t a m. (Traversable t, Applicative m) =>
 sequenceDefault tma = traverse id tma
 
 
+newtype Id a = Id a
+
+runId :: forall a. Id a -> a
+runId (Id a) = a
```

Comment:
I don't think it's that big a change, current `purescript-identity` depends on `foldable-traversable`, so it is just a swap rather than anything more complicated.